packages/gasket-cli/src/scaffold/actions/global-prompts.js

```diff
@@ -142,6 +146,9 @@ const questions = [
  * @returns {Promise} promise
  */
 async function globalPrompts(context) {
+  if (!context.prompts) {
+    inquirer.prompt = () => { return {}; };
+  }
   for (var fn of questions) {
     await fn(context);
```

Comment:
Overriding a method of a required module is kinda bad practice. What would probably be safer, and more easily testable, is to conditional pass through the prompt function to the question handlers. For example:

```suggestion
  const prompt = context.prompts ? inquirer.prompt : () => ({});
  for (var fn of questions) {
    await fn(context, prompt);
```